release-notes/formats.md

```diff
@@ -1,35 +1,47 @@
 # Release note formats
 
-Release notes are available in a combination of markdown and JSON formats. The intent is to make all release notes available in markdown and also in JSON if there is a scenario that requires structured data. This content is [licensed](./license-information.md) for broad use.
+Release notes are available in a combination of markdown and JSON formats. The intent is to make release notes available in JSON format if there is a scenario that requires structured data. This content is [licensed](./license-information.md) for broad use.
 
-Our approach has evolved over time. The following descriptions are for our latest practices.
+JSON schemas are available for JSON formats at [schemas](./schemas/README.md).
 
-## Monthly patch release notes (markdown)
+## Monthly patch release notes
 
-We typically release an update each month. These often include security fixes for vulnerabilities (AKA CVEs) disclosed on the same day.
+We typically release updates each month. These often include security fixes for vulnerabilities (AKA CVEs) disclosed on the same day. We publish release notes that provide extensive information about the release.
 
-Examples:
-
-- [6.0.32](./6.0/6.0.32/6.0.32.md)
-- [8.0.1](./8.0/8.0.1/8.0.1.md)
-
-These markdown files include the following information:
+Typical information:
 
 - Links to binaries
 - Notable changes, including CVEs
 - Compatibility information
 - Updates packages
 
-## Monthly preview release notes (markdown)
+Examples markdown files:
+
+- [6.0/6.0.32/6.0.32.md](./6.0/6.0.32/6.0.32.md)
+- [8.0/8.0.1/8.0.1.md](./8.0/8.0.1/8.0.1.md)
+
+Example JSON files:
+
+- [releases-index.json](./releases-index.json)
+- [9.0/releases.json](./9.0/releases.json)
+- [9.0/patch-release-index.json](./9.0/patch-release-index.json)
+- [Patch release overview](./9.0/preview/preview1/release.json)
 
-We typically release a preview for the next major version each month. These are much the same as the monthly patches; however, they will not include CVE information until the new major release is supported (starting with RC1).
+## Monthly preview release notes
+
+We typically release a preview for the next major version each month. These are much the same as the monthly patches.
+
+They priumarily differ in two ways:
+
+- Dense feature descriptions
+- No CVE information (since the releases are not supported)
```

Comment:
Is this still true when it gets to RC stage?

Reply:
Good catch. I had that text in there before, but it got lost. I can add it back.